cli/src/cli_util.rs

```diff
@@ -1715,7 +1715,6 @@ See https://martinvonz.github.io/jj/latest/working-copy/#stale-working-copy \
         }
 
         self.user_repo = ReadonlyUserRepo::new(tx.commit(description));
```

Comment:
nit: can you insert inline comment here?
(copied from commit description)
> update working copy before reporting repo changes, so that
> potential errors in reporting changes don't leave the repo in a stale
> state.

Reply:
btw, "leave the repo in a stale state" should be "leave the working copy ...".

Reply:
Done. Thanks.